queries.sql

```diff
@@ -141,4 +141,70 @@ FROM animals a
 JOIN owners o ON a.owner_id = o.id
 GROUP BY o.full_name
 ORDER BY animal_count DESC
+LIMIT 1;
+
+-- joint table
+
+SELECT *
+FROM animals a
+INNER JOIN visits v ON a.id = v.animal_id
+INNER JOIN vets vt ON vt.id = v.vet_id
+WHERE vt.name = 'William Tatcher'
+ORDER BY v.visit_date DESC
+LIMIT 1;
+
+SELECT COUNT(DISTINCT v.animal_id) AS animal_count
+FROM visits v
+INNER JOIN vets vt ON vt.id = v.vet_id
+WHERE vt.name = 'Stephanie Mendez';
+
+SELECT  vt.name, s.name AS specialty 
+FROM vets vt
+LEFT JOIN specializations sp ON vt.id = sp.vet_id
+LEFT JOIN species s ON sp.species_id = s.id;
+
+SELECT *
+FROM animals a
+INNER JOIN visits v ON a.id = v.animal_id
+INNER JOIN vets vt ON vt.id = v.vet_id
+WHERE vt.name = 'Stephanie Mendez'
+    AND v.visit_date BETWEEN DATE'2020-04-01' AND DATE'2020-08-30';
+
+SELECT a.name, COUNT(v.animal_id) AS visit_count
+FROM animals a
+INNER JOIN visits v ON a.id = v.animal_id
+GROUP BY a.id
+ORDER BY visit_count DESC
+LIMIT 1;
+
+SELECT *
+FROM visits v
+INNER JOIN vets vt ON vt.id = v.vet_id
+INNER JOIN animals a ON a.id = v.animal_id
+WHERE a.name = 'Maisy Smith'
+ORDER BY v.visit_date ASC
+LIMIT 1;
```

Comment:
### Who was Maisy Smith's first visit?

![image](https://github.com/Santosh-Konappanavar/Vetclinic_Database/assets/93039359/63a61db9-4b18-478d-a2a4-b38e92bd048d)


- [ ] It would be nice if you check the image of the above query 🤔 Because in the screenshot I see that your query is not returning anything but it should return the first visit of Maisy Smith and we know that there are a lot of visits done by maisy smith 👍